.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,18 @@
+<!-- Remember about a good descriptive title. -->
 
+Closes: #
+<!-- Id number of the GitHub issue this PR addresses. -->
+
+### Description
+<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->
+
+### Testing instructions
+<!-- Step by step testing instructions. When necessary break out individual scenarios that need testing, consider including a checklist for the reviewer to go through. -->
+
+### Screenshots
+<!-- Include before and after images or gifs when appropriate. -->
 
 
```

Comment:
nit: I usually separate submitter checklist from description by adding single line, otherwise it looks like a part of previous section.
```suggestion

---
```